lntest/itest/lnd_multi-hop_htlc_local_chain_claim_test.go

```diff
@@ -209,90 +178,59 @@ func testMultiHopHtlcLocalChainClaim(net *lntest.NetworkHarness, t *harnessTest,
 	// At this point we suspend Alice to make sure she'll handle the
 	// on-chain settle after a restart.
 	restartAlice, err := net.SuspendNode(alice)
-	if err != nil {
-		t.Fatalf("unable to suspend alice: %v", err)
-	}
+	require.NoError(t.t, err)
 
 	// Mine a block to confirm the two transactions (+ the coinbase).
 	block = mineBlocks(t, net, 1, expectedTxes)[0]
-	if len(block.Transactions) != expectedTxes+1 {
-		t.Fatalf("expected 3 transactions in block, got %v",
-			len(block.Transactions))
-	}
+	require.Equal(t.t, expectedTxes+1, len(block.Transactions))
```

Comment:
Nit, there's also `require.Len` (recently learned).

Reply:
Yes, I should start using that more consistently.